app/src/main/res/values/widgets.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="TheSocialNetwork.Widget" />
+
+    <style name="TheSocialNetwork.Widget.Button" parent="Widget.AppCompat.Button.Borderless">
+        <item name="android:background">@drawable/background_login</item>
```

Comment:
since we can reuse this widget, we can rename the drawable from background_login to background_button?